core/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -424,6 +438,10 @@ private void installConfig(PluginInfo info, Path tmpConfigDir, Path destConfigDi
         // create the plugin's config dir "if necessary"
         Files.createDirectories(destConfigDir);
 
+        final PosixFileAttributes destConfigDirAttributes =
+                Files.getFileAttributeView(destConfigDir.getParent(), PosixFileAttributeView.class).readAttributes();
+        setOwnerGroup(destConfigDir, destConfigDirAttributes);
```

Comment:
I'm fine with a follow up for looking at removing the necessity of this, but if we are to have user/group setting code in the plugin cli then it needs unit tests which use jimfs so they can test this behavior actually works (and can be run without needing to disable the SM).